README.md

```diff
@@ -5,18 +5,51 @@ The main goal is not to create complex functionality, but configure optimal CI/C
  
 ## Application requirements
 Install the Java 8 JDK. The java compiler is required to run the build script. 
+For Ubuntu or Debian would run:
+
+    $ sudo apt update 
+    $ sudo apt install openjdk-8-jdk
+    $ java -version
+
 Install Tomcat 8+ to run compiled WAR file.
 
-## Manual deployment
+For Ubuntu or Debian would run:
+
+    $ sudo groupadd tomcat
```

Comment:
it's worth to describe procedure with docker, as you have already had Dockerfile

Reply:
Ya, added such section